gazebo_plugins/test/test_gazebo_ros_wheelslip.cpp

```diff
@@ -0,0 +1,213 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gazebo/test/ServerFixture.hh>
+#include <gazebo_ros/node.hpp>
+#include <gazebo_ros/testing_utils.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <chrono>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+using namespace std::chrono_literals;
+
+/// Tests the gazebo_ros_wheelslip plugin
+class GazeboRosWheelSlipTest : public gazebo::ServerFixture
+{
+public:
+  void TestParameters(
```

Comment:
I think we generally follow the ROS style guide for gazebo_ros_pkgs (though the link in the contributing guide should be updated for ROS 2, see https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1333). So, I would use snake_case for method names.
```suggestion
  void test_parameters(
```

Reply:
Done